lib/modules/platform/github/readme.md

```diff
@@ -25,6 +27,26 @@ You can choose where you want to set `endpoint`:
     If you're self-hosting Renovate on GitHub.com with GitHub Actions in forking mode, and want Renovate to apply labels then you must give the PAT `triage` level rights on `issues`.
     The `triage` level allows the PAT to apply/dismiss existing labels.
 
+## Running using a Fine-grained Token
+
+### Permissions
+
+These are the permissions required for a Fine-grained token:
```

Comment:
```suggestion
A fine-grained token must have these permissions:
```

Reply:
It's a bit silly, but I can't apply my own suggestions. Can one of the maintainers apply this for me? 😉